stripe/src/main/java/com/stripe/android/view/CardMultilineWidget.java

```diff
@@ -139,8 +140,10 @@ public boolean validateAllFields() {
                 CardUtils.isValidCardNumber(mCardNumberEditText.getCardNumber());
         boolean expiryIsValid = mExpiryDateEditText.getValidDateFields() != null &&
                 mExpiryDateEditText.isDateValid();
-        boolean cvcIsValid = ViewUtils.isCvcMaximalLength(
-                mCardBrand, mCvcEditText.getText().toString());
+        int cvcLength = mCvcEditText.getText().toString().trim().length();
+        boolean cvcIsValid = cvcLength == Card.CVC_LENGTH_COMMON
```

Comment:
can we simplify this at all? This looks confusing and I'm concern it will be hard to maintain. 

One option is to pull this into a self documenting helper function and to write some tests for it because then we will be able to know what the expression is expected to resolve to